src/jit-dasm-pmi/Program.cs

```diff
@@ -387,6 +388,13 @@ void AppendEnvironmentVariableToPmiEnv(string varName, string varValue)
                     }
                 }
 
+                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    // Disable fragile relocs on non-Windows platforms, see
+                    // https://github.com/dotnet/runtime/issues/87842
+                    AddEnvironmentVariable("DOTNET_JITAllowOptionalRelocs", "0");
```

Comment:
Environment variables are case sensitive, so this should first be renamed in dotnet/runtime according to @BruceForstall's feedback.

Reply:
> Environment variables are case sensitive, so this should first be renamed in dotnet/runtime according to @BruceForstall's feedback.

Are they actually? e.g. `dotnet_jitdisasm=` seems to be working fine to me

Reply:
On Unixes they are

Reply:
ugh...

Reply:
@jakobbotsch renamed